app/views/group_offers/index.xlsx.axlsx

```diff
@@ -0,0 +1,72 @@
+wb = xlsx_package.workbook
+
+col_header = wb.styles.add_style(:bg_color => "FFDFDEDF", b: true , alignment: { horizontal: :center, vertical: :center }, border: {
+  color: '00', edges: [:bottom], style: :thin
+})
+
+wb.add_worksheet(name: t('group_offers', count: 2)) do |sheet|
+  header_row = [
+    t_attr(:title),
+    t_attr(:location),
+    t_attr(:availability),
+    t_attr(:target_group),
+    t_attr(:offer_state),
+    t_attr(:volunteers),
+    t_attr(:group_offer_category)
+  ]
+
+  sheet.add_row header_row, style: col_header
+
+  cell_types = Array.new(header_row.size).map { :string }
+  cell_styles = Array.new(header_row.size).map { nil }
+  @group_offers.each do |offer|
+    location = "#{offer.organization} #{offer.location}" if offer.external?
+    location = "#{offer.department.contact.last_name}" if !offer.external? && !offer.department.nil?
+    availabilities = ""
+    availability_collection.each do |availability|
+      if offer[availability]
+        availabilities << t("availability.#{availability}")
+        availabilities << ", "
+      end
+    end
+    targets = ""
+    if offer.all?
+      targets << t_attr(:all)
+    else
+      GroupOffer::TARGET_GROUP.each do |target|
+        if offer[target]
+          targets << t_attr(target)
+          targets << ", "
+        end
+      end
+    end
+    durations = ""
+    GroupOffer::DURATION.each do |duration|
+      if offer[duration]
+        durations << t_attr(duration)
+        durations << " "
```

Comment:
<img width="1168" alt="screen shot 2017-10-18 at 13 05 17" src="https://user-images.githubusercontent.com/11731325/31715677-1cdd1e9a-b405-11e7-99b5-bd8acb29f882.png">

![screen shot 2017-10-18 at 12 58 54](https://user-images.githubusercontent.com/11731325/31715695-24d2f0f2-b405-11e7-974d-3e224d458b6d.png)


`duration`cell is not shown eventough in one group offer its existing, thats why the content of the cells are shifted:  `volunteers` are in the `category cell` displayed

Reply:
True, I forgot to add it as a column.

Reply:
Added